lib/nostrum/struct/guild.ex

```diff
@@ -113,6 +115,18 @@ defmodule Nostrum.Struct.Guild do
   """
   @type system_channel_id :: Snowflake.t() | nil
 
+  @typedoc """
+  The id of the channel that is used for rules. This is only available to guilds that
+  contain ``PUBLIC`` in `t:features/0`.
+  """
+  @type rules_channel_id :: Snowflake.t() | nil
+
+  @typedoc """
+  The id of the channel where admins and moderators receive notices from Discord. This
+  is only available to guilds that contain ``PUBLIC`` in `t:features/0`.
+  """
+  @type public_updates_channel_id :: Snowflake.t() | nil
```

Comment:
Same as above! ^^

Reply:
OK, I changed it everywhere I could find